### ISSUE TITLE
Tool usage for MacBook

### ISSUE THREAD

**Issue by Jamil226** [andreas-mausch/whatsapp-viewer#148]
How can i use this tool for my mackbook m1. Is there any way to decrypt crypt14 file.

**Comment by 1115414801** [comment]
use this tool if you just want to decrypt crypt14 file:https://github.com/ElDavoo/WhatsApp-Crypt14-Crypt15-Decrypter

**Comment by 1115414801** [comment]
> How can i use this tool for my mackbook m1. Is there any way to decrypt crypt14 file.

use this tool if you just want to decrypt crypt14 file:https://github.com/ElDavoo/WhatsApp-Crypt14-Crypt15-Decrypter

**Comment by gusbemacbe** [comment]
> > How can i use this tool for my mackbook m1. Is there any way to decrypt crypt14 file.
> 
> use this tool if you just want to decrypt crypt14 file:https://github.com/ElDavoo/WhatsApp-Crypt14-Crypt15-Decrypter

It is exclusive of Windows.

**Comment by 1115414801** [comment]
> > > How can i use this tool for my mackbook m1. Is there any way to decrypt crypt14 file.
> > 
> > 
> > use this tool if you just want to decrypt crypt14 file:https://github.com/ElDavoo/WhatsApp-Crypt14-Crypt15-Decrypter
> 
> It is exclusive of Windows.

No,I just used it in macbook. In macbook you just need download python3.7 and then run these in your terminal:

git clone https://github.com/ElDavoo/WhatsApp-Crypt14-Crypt15-Decrypter.git
cd WhatsApp-Crypt14-Crypt15-Decrypter
python -m venv venv
source venv/bin/activate
python -m pip install --upgrade pip
python -m pip install -r requirements.txt
 
python ./decrypt14_15.py ./key ./msgstore.db.crypt14 ./msgstore.db

**Comment by Jamil226** [comment]
Thank you so much for kind help.
